ribs/emitters/operators/__init__.py

```diff
@@ -8,9 +8,15 @@
 
     ribs.emitters.operators.OperatorBase
 """
+from ribs.emitters.operators._gaussian import GaussianOperator
+from ribs.emitters.operators._iso_line import IsoLineOperator
 from ribs.emitters.operators._operator_base import OperatorBase
 
-__all__ = ["OperatorBase"]
+__all__ = [
+    "OperatorBase",
+    "GaussianOperator",
+    "IsoLineOperator",
+]
 
 _NAME_TO_OP_MAP = {}
```

Comment:
add to this map too -- probably "gaussian" and "isoline"

Reply:
I was thinking the next PR should add map key,value pairs as only the genetic algorithm emitter uses get_op.